test/suites/auth.sh

```diff
@@ -873,7 +873,7 @@ auth_project_features() {
   remote="${1}"
 
   # test-group must have no permissions to start the test.
-  [ "$(lxc query /1.0/auth/groups/test-group | jq '.permissions | length')" -eq 0 ]
+  [ "$(lxc query /1.0/auth/groups/test-group | jq -r '.permissions')" = "[]" ]
```

Comment:
I've been enjoying the use of `-e` with `jq` for assertions:

```suggestion
lxc query /1.0/auth/groups/test-group | jq -e '(.permissions | length) == 0'
```